cmd/gather-sysinfo/gather-sysinfo_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
```

Comment:
we should have moved away from ginkgo for these tests long ago

Reply:
updated